src/components/timeline/TrackContextMenu.js

```diff
@@ -5,7 +5,7 @@
 // @flow
 import React, { PureComponent } from 'react';
 import { ContextMenu, MenuItem } from 'react-contextmenu';
-import './TrackContextMenu.css';
+import 'firefox-profiler/components/timeline/TrackContextMenu.css';
```

Comment:
Please revert this line, it starts with `./` and it's not related to `src/actions`.